modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -1,27 +1,27 @@
-= Create a Secret Group (Anypoint Platform)
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Create and Edit Secret Group (Anypoint Platform)
 
+Secrets Manager enables you to create and edit secrets groups, which is a logical grouping of secrets. Secrets Manager also enables you to create new secret types and add them to a secrets group.
 
 == Prerequisites
 
+To create and edit secret groups:
+
 * Verify that you have the `Write secrets` permission enabled. 
-* Verify that you are in the correct environment using the environment switcher at the top of the left navigation bar.
+* Verify that you are in the correct environment using the environment or business group.
 
 == Create a Secret Group
 
-. In Anypoint Platform, go to Management Center and select *Secrets Manager*.
+. In Anypoint Platform, go to Management Center then select *Secrets Manager*.
 . Click *Create Secret Group*.
-. Type a name for the secret group and click *Save*.
+. Enter a name for the secret group and click *Save*.
 +
 The name of your secret group must: 
 +
-* Be at least three characters long and no longer than 35 characters
 * Start with a letter
-* Contain only letters, numbers, and dashes (but cannot end with a dash)
+* Be at least three characters long and no longer than 35 characters
+* Contain only letters, numbers, and dashes, however, the name cannot end with a dash.
 
-After creating your secret group, it appears in the *Secret Groups* list view. You can then add the necessary secret types by adding a truststore or a keystore.
+After creating your secret group, it appears in the *Secret Groups* list view. Edit the secret group to add the necessary secret types, for example adding a truststore or a keystore.
```

Comment:
```suggestion
After creating your secret group, it appears in the *Secret Groups* list view. Edit the secret group to add the necessary secret types, such as a truststore or a keystore.
```

Reply:
Take this comment with a grain of salt; adding seemed redundant, but I don't have the context to know that for sure.